docs/source/configuration_file.md

```diff
@@ -0,0 +1,139 @@
+#### random_seed
+Seed used by the random number generator to split the dataset between training/validation/testing.
+#### center_test
+List. List of centers to only include in the testing dataset.
+#### method
+Choice between `"per_patient"` (i.e. shuffle all subjects then splits) or `"per_center"` (split subject according to their acquisition centers).
+#### train_fraction
+Number between `0` and `1` representing the fraction of the dataset used as training set.
+#### test_fraction
+Number between `0` and `1` representing the fraction of the dataset used as test set. This parameter is only used if the `method` is `"per_patient"`.
+
+## Training parameters
+#### batch_size
+Integer.
+#### loss
+- `name`: Name of the loss function: Choice among the classes that are available [here](https://github.com/neuropoly/ivado-medical-imaging/blob/master/ivadomed/losses.py).
+- Other parameters that could be needed in the Loss function definition: see attributes of the Loss function of interest (e.g. `"gamma": 0.5` for `FocalLoss`).
+#### training_time
+- `num_epochs`: int.
+- `early_stopping_epsilon`: Threshold (percentage) for an improvement in the validation loss to be considered meaningful
+- `early_stopping_patience`: Number of epochs after which the training is stopped if the validation loss improvement not meaningful (i.e. less than `early_stopping_epsilon`)
+#### scheduler
+- `initial_lr`: Float. Initial learning rate.
+- `scheduler_lr`:
+1. `name`: Choice between: `"CosineAnnealingLR"`, `"CosineAnnealingWarmRestarts"` and `"CyclicLR"`. Please find documentation [here](https://pytorch.org/docs/stable/optim.html).
+2. Other parameters that are needed for the scheduler of interest (e.g. `"base_lr": 1e-5, "max_lr": 1e-2` for `"CosineAnnealingLR"`).
+#### balance_samples
+Bool. If `true`, then positive and negative GT samples are balanced in both training and validation datasets.
+#### mixup_alpha
+Float. Alpha parameter of the Beta distribution, see [original paper](https://arxiv.org/pdf/1710.09412.pdf).
+#### transfer_learning
+- `retrain_model`: Filename of the pretrained model (`path/to/pretrained-model`). If `null`, then no transfer learning is performed and the network is trained from scratch.
+- `retrain_fraction`: Float between 0. and 1. Controls the fraction of the pre-trained model that will be fine-tuned. For instance, if set to 0.5, then the second half of the model will be fine-tuned, while the first layers will be frozen.
+
+## Architecture
+Architectures for both segmentation and classification are available and described [here](models.rst).
+If the selected architecture is listed [here](https://github.com/neuropoly/ivado-medical-imaging/blob/master/ivadomed/loader/loader.py#L14), then a classification task is run, ie the ground-truth are labels extracted from `target`, instead of arrays for the segmentation task.
```

Comment:
same comment about redirecting to a file "loader" from the documentation

Reply:
Yes agree about "L14". So the idea would be to give the link to a RTD page instead of the itself code: Have I understood correctly?

Reply:
I have opened an issue [here](https://github.com/neuropoly/ivado-medical-imaging/issues/280).